drivers/hud_pit/app/models/hud_pit/generators/pit/fy2023/base.rb

```diff
@@ -0,0 +1,446 @@
+        enrollment: [
+          :disabilities,
+          :project,
+          :enrollment_coc_at_entry,
+          :health_and_dvs,
+          :exit,
+        ],
+      }
+      enrollment_scope_without_preloads.preload(preloads)
+    end
+
+    # open enrollments on the date with service that isn't extrapolated
+    private def enrollment_scope_without_preloads
+      scope = GrdaWarehouse::ServiceHistoryEnrollment.
+        entry.
+        residential.
+        ongoing(on_date: @generator.filter.on).
+        with_service_between(start_date: @generator.filter.on, end_date: @generator.filter.on, service_scope: :service_excluding_extrapolated).
+        joins(:enrollment)
+      scope = scope.in_project(@report.project_ids) if @report.project_ids.present? # for consistency with client_scope
+      scope
+    end
+
+    delegate :client_scope, to: :@generator
+
+    private def populated?
+      @report.report_cells.joins(universe_members: :client).exists?
+    end
+
+    def services_on_pit_date(batch)
+      GrdaWarehouse::ServiceHistoryService.service_excluding_extrapolated.
+        where(date: @generator.filter.on).
+        joins(:service_history_enrollment).
+        merge(enrollment_scope_without_preloads.where(client_id: batch.map(&:id))).
+        group_by(&:client_id)
+    end
+
+    private def sub_calculations
+      {
+        households: {
+          title: 'Total Number of Households',
+          query: hoh_clause,
+        },
+        clients: {
+          title: 'Total Number of Persons',
+          query: Arel.sql('1=1'),
+        },
+        veterans: {
+          title: 'Total Number of Veterans',
+          query: a_t[:veteran].eq(1),
+        },
+        children: {
+          title: 'Number of Persons (under age 18)',
+          query: child_clause,
+        },
+        youth: {
+          title: 'Number of Persons (18 - 24)',
+          query: age_ranges['18-24'],
+        },
+        age_25_34: {
+          title: 'Number of Persons (25 - 34)',
+          query: age_ranges['25-34'],
+        },
+        age_35_44: {
+          title: 'Number of Persons (35 - 44)',
+          query: age_ranges['35-44'],
+        },
+        age_45_54: {
+          title: 'Number of Persons (45 - 54)',
+          query: age_ranges['45-54'],
+        },
+        age_55_64: {
+          title: 'Number of Persons (55 - 64)',
+          query: age_ranges['55-64'],
+        },
+        age_over_64: {
+          title: 'Number of Persons (over age 64)',
+          query: age_ranges['64+'],
+        },
+        youth_hoh: {
+          title: 'Number of parenting youth (youth parents only)',
+          query: hoh_clause.and(age_ranges['18-24']),
+        },
+        child_hoh: {
+          title: 'Number of parenting youth (under age 18)',
+          query: hoh_clause.and(child_clause),
+        },
+        hoh_for_youth: { # note because the question is already limted to youth households, this is just here to provide the title
```

Comment:
```suggestion
        hoh_for_youth: { # note because the question is already limited to youth households, this is just here to provide the title
```